source/io/module_httpClient.js

```diff
@@ -625,6 +629,19 @@
                 Module.eggShell.setOccurrenceAsync(occurrencePointer);
             });
         };
+
+        Module.httpClient.jsHttpClientSetCredentials = function (handle, isWithCredentials, errorStatusPointer, errorCodePointer, errorSourcePointer) {
+            var errorStatus = Module.eggShell.dataReadBoolean(errorStatusPointer);
```

Comment:
Sorry, looks like you do have the new pattern on the C++ side.

That means you can remove this errorStatus check on the js side altogether.